app/helpers/application_helper.rb

```diff
@@ -1,2 +1,7 @@
 module ApplicationHelper
+
+  def pretty_date(date)
+    ("<span class='date'>" + date.strftime('%a %d %b %Y') + "</span>").html_safe
```

Comment:
Need to add a comma or two to the date string (possibly change format to American date? example: Oct 23, 2017.)